### ISSUE TITLE
Fix INDE boundary

### ISSUE THREAD

**Issue by nateirwin** [nationalparkservice/places-data#552]
![park_tiles_3___insidemaps](https://cloud.githubusercontent.com/assets/10443/13155794/a81eda96-d63c-11e5-90d1-76fe138ad8c0.png)

**Comment by chadlawlis** [comment]
That's actually correct - as seen in Lands:

<img width="613" alt="inde" src="https://cloud.githubusercontent.com/assets/1238636/13156091/0a3b6338-d63e-11e5-8a1f-3ab3f09733b8.png">

Currently pulling from WSD_Parks, however, so we can transition this to Lands.

**Comment by nateirwin** [comment]
Color me skeptical ;-)

**Comment by chadlawlis** [comment]
Hm, agreed: http://www.nps.gov/hfc/carto/PDF/INDEmap1.pdf.

Reopening - will ping Lands and the park. Their web author reached out via WSD feedback so I can get their take on it.

**Comment by chadlawlis** [comment]
I pinged Todd, Park Ranger @ INDE about this.

**Comment by chadlawlis** [comment]
From Todd:

> I noticed that and was going to ask you guys about it.  So I'm glad you brought it up.  
> Is that green shaded area with the dark green borders supposed to represent INDE grounds?  If it is, it definitely needs to be adjusted so that it matches up with the HFC map, which from the looks of it, might mean some major adjustments for the whole thing.  For example:  That whole southern section of the park is off by entire block.  It should be between Chestnut and Walnut Streets, not Walnut and Spruce Streets, and does not extend all the way to Front Street.  
> The INDE HFC map is accurate and pretty up to date, so I would use that as a guideline for the boundaries of the park.  The only difference is that the Living History Center doesn't exist anymore.  The Museum of the American Revolution is in the process of being built at that location. 

I copied Lands on my response. This will require a new boundary from the park/HFC or a Lands update to fix as we have no alternative on file.

**Comment by chadlawlis** [comment]
From Nigel, 2/29/16:

> The authorized INDE boundary is in draft and just needs to be pushed along through the steps - what [Lands] is working on now.

From Nigel, 3/2/16:

> Mark said the INDE Segment 01 needs some work which he hopes to get to  before the end of next week.

**Comment by chadlawlis** [comment]
This is still in draft/review via NER Lands. Check back in with Nigel and Lands as the next Lands release approaches (presumably around the end of September).

**Comment by chadlawlis** [comment]
Updated boundary provided by NER Lands, available [here](https://drive.google.com/open?id=1wp3CjwbvVMbKdOrTGis1HlV8yVjTAEP3).

**Comment by chadlawlis** [comment]
Updated in DB via:

```sql
UPDATE parks_poly SET geom_poly = (SELECT ST_Union(wkb_geometry) FROM inde) WHERE unit_id = 205;

UPDATE parks SET unit_area = (SELECT ST_Area(geom_poly) FROM parks_poly WHERE unit_id = 205) WHERE unit_id = 205;
```

Will go live once boundaries are regenerated.

**Comment by chadlawlis** [comment]
This is now live.

**Comment by chadlawlis** [comment]
The same geometry from NER Lands is now available in the official Lands release from 20171231 (https://github.com/nationalparkservice/places-data/issues/803).

Updating the `data_source` in the DB to "Land Resources Division" (replacing "ner").
